src/core/alien/kernels/redistributor/Redistributor.cc

```diff
@@ -43,12 +45,17 @@ namespace Alien
 Redistributor::Redistributor(
 int globalSize, Arccore::MessagePassing::IMessagePassingMng* super, bool keep)
 : m_super_pm(super)
+, m_target_comm(Arccore::MessagePassing::mpSplit(super, keep))
```

Comment:
This `IMessagePassingMng` is owned by the `Redistributor` and should not be used after this object is deleted.

I think it is a dangerous behavior and that we should mark this ctor has deprecated.

Any thought about this @gratienj and @NathMoller ?

Reply:
We have chosen to remove this ctor.